bugsnag-android-core/src/main/java/com/bugsnag/android/ThreadState.kt

```diff
@@ -109,13 +109,13 @@ internal class ThreadState @Suppress("LongParameterList") constructor(
             // API 24/25 don't record the currentThread, so add it in manually
             // https://issuetracker.google.com/issues/64122757
             // currentThread may also have been removed if its ID occurred after maxThreadCount
-            keepThreads.take(Math.max(maxThreadCount - 1, 0)).plus(currentThread).sortedBy { it.id }
+            keepThreads.take((maxThreadCount - 1).coerceAtLeast(0)).plus(currentThread).sortedBy { it.id }
```

Comment:
nit: an `import kotlin.math.max` allows:

```suggestion
            keepThreads.take(max(maxThreadCount - 1, 0)).plus(currentThread).sortedBy { it.id }
```